Extend timeout to avoid failing tests due to temporarily unresponsive storage
Author: Samuel Moors, Vrije Universiteit Brussel (VUB)
diff -ur libxc-4.3.4.orig/testsuite/CMakeLists.txt libxc-4.3.4/testsuite/CMakeLists.txt
--- libxc-4.3.4.orig/testsuite/CMakeLists.txt	2019-01-25 15:47:40.000000000 +0100
+++ libxc-4.3.4/testsuite/CMakeLists.txt	2020-08-13 23:34:08.091740000 +0200
@@ -42,7 +42,7 @@
         set(test "${func}-${system}-${spin}-${order}")
         add_test(NAME ${test}-run
                  COMMAND xc-regression ${func} ${nspin} ${order} ${CMAKE_CURRENT_SOURCE_DIR}/input/${system} ${CMAKE_CURRENT_BINARY_DIR}/workdir/${_sdir}/${refname})
-        set_tests_properties(${test}-run PROPERTIES TIMEOUT 1)
+        set_tests_properties(${test}-run PROPERTIES TIMEOUT 20)
 
         # checke the tests values against the reference values
         add_test(NAME ${test}-check
